skycultures/greek_farnese/info.ini

```diff
@@ -0,0 +1,7 @@
+[info]
+name = Greek (Farnese + Almagest)
```

Comment:
I guess this name is not actual now...

Reply:
This name is OK I think, and describes this as a mixed issue. The globe is described as Roman replica of a lost Greek artwork showing constellations following Aratos, and the additional description and stick figures are from Almagest. We may want to open a new category "comparative" for this, though, as all the additions over the Farnese globe exceed what a "single" SC is allowed to provide. This category can also be given to the mix of Aratea+AlmagestStickFigures to make it clearer why some stick figures do not match the paintings.

Reply:
ah! QUESTION I updated the addition in the Leiden SC. Could you please tell me what you think?! 
Initially, I had "Leiden Aratea + Almagest" and "Farnese + Alm." but the first appeared too long in my display window which is the reason why I deleted "Alm." before uploading. One of you pointed out correctly that the mixture of stickfigures and artwork from different cultures is not "single". That is why, I have now written it again. 

Background: Initially, I wanted to provide original artwork for the almagest SC that is the only one which is exactly (point by point) defined. However, all artwork is based on Aratus and, thus, does not fit perfectly the mathematical tradition in which the Almagest is written. That's why I think, mentioning both would be best. That is from the content-point of view. 

But the display-point of view? Is it ok for you or do you think, it should be abbreviated or simplified anyhow?

Reply:
We could widen the list box somewhat to avoid the "..." display. (Is there some auto-width possible?)

Aratus laid the foundation of the Greek tradition which was taken up, enriched by a few more figures and cast into numbers by Ptolemy. Presumably astronomers/intrument-makers were able to make globes with stars at the right places (and some frame/mount for actual operations, so presumably such globe was the common astronomical computer/display that exceeded the planispheric astrolabe), and then the artisan optionally created figures similar to those we still know, but most ancient globes are lost. (Wood can burn, metal could be used for guns... :-( Medieval scribes apparently neither observed the sky nor used coordinate lists to make what we call "maps" today but just copied what they found in older manuscripts, sometimes mixing up similar figures. The Leiden Aratea (which BTW has Germanicus' text, yet another deviation to mention?) is thus no atlas in the modern sense but a picture book, the figures of which can as it seems be placed into the sky only with major compromises in accepting distortions. Another "picture book" with merely symbolic stars in its constellation drawings is the Ratdolt Hyginus (and a few following editions in the 16th century). AFAIK, apart from the Dürer planispheric map and e.g. Mercator's 1551 globe (and an earlier one from Gemma Frisius(?)) which are also based on the Almagest data, the earliest printed coordinate-based star atlas is the 1561 Piccolomini, which then has no figures, so that Bayer's is likely the first which has Ptolemy's coordinates and figures. 

This said, is the Farnese globe the only one showing only Aratus' and not Ptolemy's sky with a claim to have the coordinates (at least somewhat) right?

Reply:
> We could widen the list box somewhat to avoid the "..." display. (Is there some auto-width possible?)

I've added tooltips for list of skycultures

Reply:
> We may want to open a new category "comparative" for this, though, as all the additions over the Farnese globe exceed what a "single" SC is allowed to provide.

Good proposal, but the rules for new category should be clear and probably strong

Reply:
yes, great idea, thanks!